### ISSUE TITLE
Scaffolder Log Stream SSE Not Compatible with Ingress Compression

### ISSUE THREAD

**Issue by OscarDHdz** [backstage/backstage#5169]
## Expected Behavior

Scaffolder logs stream should display when Ingress compression is on

## Current Behavior

Task page from scaffolder just show a "Loading..." log as SSE never reach out the client browser.

<img width="671" alt="Screen Shot 2021-03-29 at 12 30 45" src="https://user-images.githubusercontent.com/18434722/112882754-984cf900-908a-11eb-9eac-41cd5b1a68e6.png">

## Context

At Expedia, we upgraded our core backstage dependencies a few weeks ago, and since then we noticed the Scaffolder Logs stream stopped to show. After doing some research, we found that the issue was that SSEs were never reaching out the client browser due some compression happening in our infrastructure.
We believe that this issue might happen to other adopters, and that's why we are raising this issue to start a conversation.

**Comment by freben** [comment]
Interesting! Spontaneously it sounds like it should be possible to fix with request header tweaks - this seems to suggest the same https://stackoverflow.com/questions/23769001/is-it-possible-to-use-gzip-compression-with-server-sent-events-sse

**Comment by benjdlambert** [comment]
@OscarDHdz is there something that we're missing from the core that we can contribute back or more of a documenting gotcha thing?

**Comment by OscarDHdz** [comment]
Sorry for the late response, we were trying different things to get around this.

In the end, we had to add a rule to our ingress controller  to bypass compression for these Server Side Events. We don't see a way to contribute back  😢 We did had in mind to add some sort of flag to toggle between SSE/SimpleHTTP/Socket but that might had been too much for the root cause (adopter infrastructure). 

Closing this Issue 😄
